app/views/pages/derivatives.html.erb

```diff
@@ -0,0 +1,160 @@
+                </ul>
+              </td>
+            </tr>
+            <tr class="tr_table">
+              <td class="td_table"><strong>File Output</strong></td>
+              <td class="td_table"><kbd>.txt</kbd></td>
+            </tr>
+            <tr class="tr_table">
+              <td class="td_table"><strong>Files Generated</strong></td>
+              <td class="td_table">A <strong>text file</strong> is generated which can be used in various other programs and software.</td>
+            </tr>
+            <tr class="tr_table">
+              <td class="td_table"><strong>Additional Resources</strong></td>
+              <td class="td_table">
+                <p class="about_p">Working with lots of text can be challenging. Fortunately, there are lots of resouces on the web to help teach you the basics of text analysis.</p>
+                <ul>
+                  <li class="card_li"><strong><a href="https://programminghistorian.org">Programming Historian</a></strong>: The Programming Historian has many lessons, including a specific section on <a href="https://programminghistorian.org/en/lessons/?topic=distant-reading">distant reading."</a></li>
+                  <li class="card_li"><strong><a href="http://tapor.ca/home">Text Analysis Portal for Research</a></strong>: TAPoR, or the Text Analysis Portal for Research, provides a curated collection of tools that can be used to explore text.</li>
+                  <li class="card_li"><strong><a href="https://uottawa.libguides.com/DH/textual">Digital Humanities Guide at University of Ottawa</a></strong>: This is a great guide that highlights several approaches to text analysis in the field.</li>
+                  <li class="card_li"><strong><a href="http://voyant-tools.org/">Voyant Tools</a></strong>: This is more of a tool than an introductory resource, but it's a great "gateway drug" into the powers of text analyis.</li>
+                </ul>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+  <div class="accord">
+    <div class="accord_header" id="headingThree">
+      <h5 class="mb-0">
+        <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapseThree" aria-expanded="false" aria-controls="collapseThree">Domains File
+        </button>
+      </h5>
+    </div>
+    <div id="collapseThree" class="collapse" aria-labelledby="headingThree" data-parent="#accordion">
+      <div class="accord_body">
+        <p class="about_p">Sometimes you might just want to know what is inside a web archive! The web is a complex beast: a small web archive that aims to collect just one page might end up capturing embedded widgets and content from other websites just to make something play; alternatively, a crawl might have been corrupted and begun to find material that's far out of scope.</p>
```

Comment:
Heh, I probably added the semicolon. Let's remove the semicolon and make the last two sentences separate. like:

```
play. Alternatively, a crawl might have been corrupted and begun to find material that's far out of scope.
```